tests/fullydistributed_grids/copy_distributed_tria_01.with_p4est=true.with_metis=true.mpirun=1.output

```diff
@@ -0,0 +1 @@
+
```

Comment:
why do you need metis?

Reply:
You are right! These are historical remainders...